contrib/test_jenkins.sh

```diff
@@ -1014,6 +1014,58 @@ test_memtrack() {
 	UCX_MEMTRACK_DEST=stdout UCX_HANDLE_ERRORS=none UCX_MEMTRACK_LIMIT=412MB ./test/apps/test_memtrack_limit |& grep -C 100 'reached'
 }
 
+test_namespace() {
+	perftest="$ucx_inst/bin/ucx_perftest"
+	info="./src/tools/info/ucx_info"
+
```

Comment:
pls note that alignment is wrong (this file uses tabs)

Reply:
fixed thanks, will force push